tests/loan-token/LendingTestToken.test.js

```diff
@@ -118,6 +122,7 @@ contract("LoanTokenLending", (accounts) => {
         /** Initialize the loan token logic proxy */
         loanToken = await ILoanTokenLogicProxy.at(loanToken.address);
         await loanToken.setBeaconAddress(loanTokenLogicBeacon.address);
+        expect(await loanToken.getTarget()).to.equal(loanTokenLogic.address);
```

Comment:
pls move it into a separate test  
best practice is not to have it initialization scripts